s3.tf

```diff
@@ -17,7 +17,7 @@ data "template_file" "bucket_policy" {
   vars = {
     #if apiary_shared_schemas is empty or contains current schema, allow customer accounts to access this bucket.
     customer_principal = "${length(var.apiary_shared_schemas) == 0 || contains(var.apiary_shared_schemas, each.key) ?
-    join("\",\"", formatlist("arn:aws:iam::%s:root", var.apiary_customer_accounts)) : ""}"
+    join("\",\"", formatlist("arn:aws:iam::%s:root", split(",", each.value["customer_accounts"]))) : ""}"
```

Comment:
If someone wants to disable customer-account access for a particular schema, would it work to set `customer_accounts = ""`?  It appears as if it would, which would also be a nice feature.

Reply:
yes that should work!